modules/ble/src/main/java/com/gluonhq/attach/ble/impl/AndroidBleService.java

```diff
@@ -60,22 +60,17 @@
     private static final ObservableList<BleDevice> devices = FXCollections.observableArrayList();
     private static final List<String> deviceNames = new LinkedList<>();
     private static final List<String> profileNames = new LinkedList<>();
-    private static boolean debug;
+    private static final boolean debug = Boolean.getBoolean(Constants.ATTACH_DEBUG);
```

Comment:
The static debug variables that are in a lot of services can be replaced by an instance variable and be assigned with `Util.isDebug()`? Or you could even just drop the variable entirely and use `Util.isDebug()` directly in the if statements?

Reply:
Same, different Util (dalvik). We would have to add a public method in the Util module for this. `AndroidDebug` is not accessible to other modules.

Reply:
I see. It does make sense to add that public method in the util module, because I can see it's used everywhere.

Reply:
Makes sense, done